angular-rails-app/lib/tasks/db_connector.rake

```diff
@@ -49,32 +49,39 @@ namespace :db_connector do
 
     conn = ActiveRecord::Base.connection
 
-    # Load addresses from Pluto
-    sql = "INSERT IGNORE INTO r_properties (street_address,city,state,zipcode,total_units,borough,block,lot,created_at,updated_at)
-    SELECT TRIM(address), 'New York', 'New York', zipcode, unitstotal, borough, block, lot, NOW(), NOW() FROM pluto_nyc;"
-    records_array = conn.execute(sql)
-
-    @boros.each do |key, value|
-      sql = "UPDATE hpd_buildings SET boro = '#{key}' WHERE boro = '#{value}';"
-      conn.execute(sql)
-    end
-
-    # Load addresses from HPD. This will catch additional addresses in hpd that are *not* in pluto
+    # Load addresses from HPD.
+    print "Copying from hpd_buildings..."
     sql = "INSERT IGNORE INTO r_properties (street_address,city,state,zipcode,hpd_registration_id,borough,block,lot,created_at,updated_at)
-    SELECT TRIM(streetname), 'New York', 'New York', zip, registrationid, boro, block, lot, NOW(), NOW() FROM hpd_buildings;"
+    SELECT TRIM(streetname), 'New York', 'New York', zip, registrationid, boro, block, lot, NOW(), NOW() FROM hpd_buildings WHERE streetname != '' AND streetname IS NOT NULL AND registrationid != 0;"
```

Comment:
"New York" does not seem like it would always be the right city, given boroughs are used for postal addresses...?